.gitignore

```diff
@@ -72,3 +72,4 @@ vendor/bundle/
 .swiftpm
 Example/Carthage/.env
 Example/Carthage/fastlane/report.xml
+Sample/Cocoapods/Podfile.lock
```

Comment:
Why this?
Also, we should probably update the `Example/` paths above to match the new structure, too.

Reply:
I've tried to make sure Cocoapods example always builds the podspec swift version, but seems like Xcode treats all swift 5 versions equally and cannot build with old versions. So in my little experiment, `pod install` created the podfile, but IMHO the podfile.lock should never be checked in, since it'd mess with CI and stuff

Reply:
OK, for samples app, we always want the latest versions.